docassemble/GithubFeedbackForm/data/questions/feedback.yml

```diff
@@ -279,17 +281,20 @@ fields:
     field: share_other_details   
     datatype: area
     required: False
-#  - label: |
-#      **Share my specific answers with an administrator**[BR]
-#      You can optionally share your answers on the online form with an
-#      administrator. Only two Suffolk University employees can view
-#      this information. It will not be made public.
-#      [BR]
-#      If you say no, it may be harder for us to track down the problem,
-#      but we will still try our best.
-#      We will not contact you either way.
-#    field: share_interview_answers
-#    datatype: yesnoradio
+  - label: |
+      **Share my specific answers with an administrator**[BR]
+      You can optionally share your answers on the online form with an
+      administrator. Only server administrators can view
+      this information. It will not be made public.
+      [BR]
+      If you say no, it may be harder for us to track down the problem,
+      but we will still try our best.
+      We will not contact you either way.
```

Comment:
Just some small thoughts on language. You don't have to take the whole thing, it's just here if you want it. I feel that if we want the sentiment of "we will not contact you" for the whole thing, that should be in a different place.

```
**Share my answers with an administrators**[BR]
Your answers can help us try to track down the problem. Only server administrators will be able to see your answers.
```

@CaroRob , any thoughts for language on this page?

Reply:
Thanks! I had made my own version at https://github.com/SuffolkLITLab/docassemble-GithubFeedbackForm/pull/22/files#diff-513abd93f534244b9f400e7681d5fa5ad0b8093562cba85b9cced5dfde618f23R103-R105, which is 

> If you ran into a bug, you can share your answers with
      an administrator (they will not be made public) to help us
      track down the problem.
      (label was next to field, not above)
      
  But I like parts of this as well. Might combine the two:
  
  > **Share my answers with administrators**[BR]
If you ran into a bug, your answers can help us track down the problem. Your answers will not be made public.